### PR TITLE
Added support for host entry in docker container

### PR DESCRIPTION
**Pull request by bdpiparva**
* User can map external host entry in docker container
Host entry must be in `IP-ADDRESS <space/tab> HOSTNAME-1 <space/tab>
HOSTNAME-2...` format

Example
```host
10.0.0.1  host-x
10.0.0.2  host-y  host-z
```

### CODE REVIEW

File: src/main/java/cd/go/contrib/elasticagents/dockerswarm/elasticagent/Hosts.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+            hostMappings.addAll(toHosts(hostEntry));
+        });
+
+        return hostMappings;
+    }
+
+    private List<String> toHosts(String hostEntry) {
+        String[] parts = hostEntry.split("\\s+");
+
+        final String ipAddess = parts[0];
```
Comment (ketan) [inline src/main/java/cd/go/contrib/elasticagents/dockerswarm/elasticagent/Hosts.java]: Typo?

File: src/main/resources/profile.template.html [hunk 2]
```diff
@@ -13,43 +13,124 @@
...
-    <textarea ng-class="{'is-invalid-input': GOINPUTNAME[Environment].$error.server}" type="text" ng-model="Environment" ng-required="true" rows="7" placeholder="JAVA_HOME=/opt/java&#x000A;MAVEN_HOME=/opt/maven"></textarea>
-    <span class="form_error form-error" ng-class="{'is-visible': GOINPUTNAME[Environment].$error.server}" ng-show="GOINPUTNAME[Environment].$error.server">{{GOINPUTNAME[Environment].$error.server}}</span>
+
+  <div class="row">
+    <div class="columns end">
+      <label ng-class="{'is-invalid-label': GOINPUTNAME[Hosts].$error.server}">Host entries
+        <div class="tooltip-info">
+          <span class="tooltip-content">
+            This allows users to add host entries in <code>/etc/hosts</code>.
+            <div>Enter each host entry per line and in below mentioned format:
```
Comment (ketan) [inline src/main/resources/profile.template.html]: `and in the following format:`

**Comment by tpbrown** [comment]
Any plans to support named overlay networks in addition to this?

e.g. - docker network create -d overlay pipeline, docker service create --name go-server --network pipeline, and then you'd want all services/agents created to also be on the 'pipeline' network.

**Comment by ketan** [comment]
@tpbrown - do you want to submit a new issue for this. The implementation is fairly straightforward if you want to implement it

**Pull request merged**